views/admin-page-settings.php

```diff
@@ -28,42 +26,15 @@
 					<?php esc_html_e( 'Your pages', 'progress-planner' ); ?>
 				</h2> -->
 				<div class="prpl-pages-list">
-					<?php foreach ( $prpl_tabs_settings as $prpl_tab_key => $prpl_tab ) : ?>
-
-						<div>
-							<?php /* phpcs:disable */ ?>
-							<?php /* echo esc_html( $tab['title'] ); */ ?>
-							<?php /* echo esc_html( $tab['desc'] ); */ ?>
-
-							<?php foreach ( $prpl_tab['settings'] as $prpl_setting ) : ?>
-
-								<?php
-									// WIP. Skip radio buttons with page-select.
-								if ( false !== strpos( $prpl_setting['id'], 'has-page-' ) ) {
-									continue;
-								}
-								?>
-
-								<div class="prpl-pages-item-setting">
-									<?php if ( 'page-select' === $prpl_setting['type'] ) : ?>
-										<div
-											class="prpl-pages-item prpl-pages-item-<?php echo esc_attr( $prpl_setting['page'] ); ?>"
-											data-page-item="<?php echo esc_attr( $prpl_setting['page'] ); ?>"
-										>
-											<?php \progress_planner()->the_view( "setting/{$prpl_setting['type']}.php", [ 'prpl_setting' => $prpl_setting ] ); ?>
-										</div>
-									<?php else : ?>
-										<div class="prpl-pages-item">
-											<?php /* phpcs:disable */ ?>
-											<?php /* echo isset( $prpl_setting['title'] ) ? esc_html( $prpl_setting['title'] ) : ''; */ ?>
-											<?php /* echo isset( $prpl_setting['label'] ) ? esc_html( $prpl_setting['label'] ) : ''; */ ?>
-											<?php /* phpcs:enable */ ?>
-											<?php \progress_planner()->the_view( "setting/{$prpl_setting['type']}.php", [ 'prpl_setting' => $prpl_setting ] ); ?>
-										</div>
-									<?php endif; ?>
-								</div>
-							<?php endforeach; ?>
-						</div>
+					<?php foreach ( \progress_planner()->get_admin__page_settings()->get_tabs_settings() as $prpl_tab_key => $prpl_tab ) : ?>
+						<?php foreach ( $prpl_tab['settings'] as $prpl_setting ) : ?>
+							<div
+								class="prpl-pages-item prpl-pages-item-<?php echo esc_attr( $prpl_setting['page'] ); ?>"
+								data-page-item="<?php echo esc_attr( $prpl_setting['page'] ); ?>"
+							>
```

Comment:
@ilicfilip Can you please move this wrapper `div` to the `views/setting/page-select.php` file?
This will make it easier for us to add more settings with different templates from the PRO plugin 🙇

Reply:
moved!